gnome-extra/cinnamon/cinnamon-4.4.8.ebuild

```diff
@@ -0,0 +1,193 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+GNOME2_LA_PUNT="yes"
+PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_REQ_USE="xml"
+
+inherit autotools eutils flag-o-matic gnome2 multilib pax-utils python-single-r1
+
+DESCRIPTION="A fork of GNOME Shell with layout similar to GNOME 2"
+HOMEPAGE="http://developer.linuxmint.com/projects/cinnamon-projects.html"
+
+MY_PV="${PV/_p/-UP}"
+MY_P="${PN}-${MY_PV}"
+
+SRC_URI="https://github.com/linuxmint/cinnamon/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+
+IUSE="+nls +networkmanager"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+KEYWORDS="~amd64 ~x86"
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	app-accessibility/at-spi2-atk:2
+	app-misc/ca-certificates
+	dev-libs/dbus-glib
+	>=dev-libs/glib-2.35.0:2[dbus]
+	>=dev-libs/gobject-introspection-1.29.15:=
+	>=dev-libs/json-glib-0.13.2
+	>=dev-libs/libcroco-0.6.2:0.6
+	dev-libs/libxml2:2
+	gnome-base/librsvg
+	>=gnome-extra/cinnamon-desktop-4.0:0=[introspection]
+	>=gnome-extra/cinnamon-menus-4.4
+	>=gnome-extra/cjs-4.0.0[cairo]
+	>=media-libs/clutter-1.10:1.0[introspection]
+	media-libs/cogl:1.0=[introspection]
```

Comment:
These two aren't needed, because cinnamon is using cogl and clutter bundled with and provided by the muffin... https://github.com/linuxmint/cinnamon/blob/master/configure.ac#L84-L87